### PR TITLE
Update tiup-component-cluster-scale-in.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add a warning note

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-component-cluster-scale-in.md [hunk 1]
```diff
@@ -39,9 +39,9 @@ tiup cluster scale-in <cluster-name> [flags]
 - 数据类型：`BOOLEAN`
 - 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
 
-> **注意：**
+> **警告：**
 >
-> 强制移除 TiKV 节点不会等待数据调度，移除一个以上正在提供服务的 TiKV 节点会有数据丢失的风险。
+> 使用该命令强制移除正在服务的 TiKV / TiFlash 节点时，这些节点会直接下线，不进行数据调度，因此这个场景下存在较大概率丢失数据的风险。官方不建议对非宕机的节点使用该命令。
```
Comment (TomShawn) [inline tiup/tiup-component-cluster-scale-in.md:44]: 是针对该命令还是该选项呢？

**Comment by TomShawn** [comment]
@lilyjazz 能请一两位研发老师看看吗？谢谢

File: tiup/tiup-component-cluster-scale-in.md (hunk 1 above)
Comment (lilyjazz) [reply tiup/tiup-component-cluster-scale-in.md:44]: 应该是选项。我改一下

File: tiup/tiup-component-cluster-scale-in.md [hunk 2]
```diff
@@ -39,9 +39,9 @@ tiup cluster scale-in <cluster-name> [flags]
 - 数据类型：`BOOLEAN`
 - 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
 
-> **注意：**
+> **警告：**
 >
-> 强制移除 TiKV 节点不会等待数据调度，移除一个以上正在提供服务的 TiKV 节点会有数据丢失的风险。
+> 使用该选项强制移除正在服务的 TiKV / TiFlash 节点时，这些节点会直接下线，不进行数据调度，因此这个场景下存在较大概率丢失数据的风险。官方不建议对非宕机的节点使用该选项。
```
Comment (AstroProfundis) [inline tiup/tiup-component-cluster-scale-in.md:44]:
```suggestion
> 使用该选项强制移除正在服务和下线中的 TiKV / TiFlash 节点时，这些节点会被直接删除，不等待数据调度完成，因此这个场景下存在较大概率丢失数据的风险。官方不建议对未宕机的节点使用该选项。
```

**Review by AstroProfundis — APPROVED** [review]

**Review by qqqdan — APPROVED** [review]

File: tiup/tiup-component-cluster-scale-in.md [hunk 3]
```diff
@@ -39,9 +39,9 @@ tiup cluster scale-in <cluster-name> [flags]
 - 数据类型：`BOOLEAN`
 - 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
 
-> **注意：**
+> **警告：**
 >
-> 强制移除 TiKV 节点不会等待数据调度，移除一个以上正在提供服务的 TiKV 节点会有数据丢失的风险。
+> 使用该选项强制移除正在服务和下线中的 TiKV / TiFlash 节点时，这些节点会被直接删除，不等待数据调度完成，因此这个场景下存在较大概率丢失数据的风险。官方不建议对未宕机的节点使用该选项。
```
Comment (TomShawn) [inline tiup/tiup-component-cluster-scale-in.md:44]:
```suggestion
> 使用该选项强制移除正在服务和下线中的 TiKV / TiFlash 节点时，这些节点会被直接删除，不等待数据调度完成，因此这个场景下存在较大概率丢失数据的风险。不建议对未宕机的节点使用该选项。
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**